tests/Application/Actions/Donations/UpdateTest.php

```diff
@@ -1896,4 +1875,82 @@ public function testAddDataAutoconfirmHitsUnknownInvalidRequestException(): void
         // act
         $app->handle($request->withAttribute('route', $route));
     }
+
+    /**
+     * @return array{
+     *     app: App,
+     * request: ServerRequestInterface,
+     * route: Route,
+     * donationRepoProphecy: ObjectProphecy<DonationRepository>,
+     * entityManagerProphecy: ObjectProphecy<EntityManagerInterface>,
+     * stripePaymentIntentsProphecy: ObjectProphecy<PaymentIntentService>
+     * }
+     *
+     * @throws \Doctrine\DBAL\Exception\LockWaitTimeoutException
+     * @throws \MatchBot\Application\Matching\TerminalLockException
+     * @throws \MatchBot\Domain\DomainException\DomainLockContentionException
+     * @throws \Stripe\Exception\ApiErrorException
+     */
+    public function setupTestDoublesForConfirmingPaymentFromDonationFunds(string $newPayemtnIntentStatus): array
```

Comment:
```suggestion
    public function setupTestDoublesForConfirmingPaymentFromDonationFunds(string $newPaymentIntentStatus): array
```
and refs to that var